src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -95,14 +95,19 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
         }, [fetchData]),
     );
 
-    const createNewApprovalWorkflow = useCallback(() => {
-        Workflow.setApprovalWorkflow({
-            ...EMPTY_APPROVAL_WORKFLOW,
-            availableMembers: approvalWorkflows.at(0)?.members ?? [],
-        });
-
-        Navigation.navigate(ROUTES.WORKSPACE_WORKFLOWS_APPROVALS_EXPENSES_FROM.getRoute(route.params.policyID));
-    }, [approvalWorkflows, route.params.policyID]);
+    const addApprovalAction = useCallback(() => {
+        if (!PolicyUtils.isControlPolicy(policy)) {
```

Comment:
Please add a comment to the code that explains why these two logic paths are necessary?